components.html

```diff
@@ -0,0 +1,1473 @@
+            <svg class="migration-free-advert-icon" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+              <g fill="none" fill-rule="evenodd" transform="translate(3 3)">
+                <circle cx="9" cy="9" fill="#0777ef" r="9" />
+                <path d="m5 9.5 3.2 2.5 4.8-6" stroke="#fff" stroke-width="2" />
+              </g>
+            </svg>
+
+            <p>Priority email support</p>
+          </li>
+          <button class="btn migration-free-advert-btn">Find out more</button>
+        </ul>
+
+      </div>
+
+      <div class="column align-right " style="margin-bottom:auto; flex:1">
+
+        <iframe allowFullScreen="allowFullScreen" src="https://www.youtube.com/embed/8u_kX5d78Bs?ecver=1&amp;iv_load_policy=1&amp;yt:stretch=16:9&amp;autohide=1&amp;color=red&amp;width=560&amp;width=560" width="480" height="270"
+          allowtransparency="true" frameborder="0">
+          <script>
+            function execute_YTvideo() {
+              return youtube.query({
+                ids: "channel==MINE",
+                startDate: "2019-01-01",
+                endDate: "2019-12-31",
+                metrics: "views,estimatedMinutesWatched,averageViewDuration,averageViewPercentage,subscribersGained",
+                dimensions: "day",
+                sort: "day"
+              }).then(function(e) {}, function(e) {
+                console.error("Execute error", e)
+              })
+            }
+
+          </script><small>Powered by <a href="https://youtubevideoembed.com/ ">Embed YouTube Video</a></small>
+        </iframe>
+
+        <div class="flex-container tweet-testimonial">
+          <div class="column">
+            <img class="avatar" src="./img/testimonial-avatar.png">
+          </div>
+
+          <div class="column">
+            <div class="flex-container" style="flex-grow:1; align-items: center;">
+              <h4 class="text-white full-opacity">MortenRandHendriksen</h4>
+              <p class="twitter-handle text-white">@mor10</p>
+
+              <div class="align-right">
+                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+                  <path fill="#A5DDF1"
+                    d="M8.57952685,1.37620917 L10.2429656,4.90369756 C10.3369312,5.10295441 10.5185555,5.24105328 10.7286459,5.27298678 L14.4476796,5.83873515 C14.6907017,5.87567445 14.8925991,6.05385549 14.968446,6.29832852 C15.0442929,6.54280156 14.9809285,6.8111458 14.8050061,6.99048543 L12.1134474,9.73618788 C11.9612289,9.89146488 11.8922145,10.1149287 11.9276892,10.3336668 L12.563008,14.210866 C12.6044673,14.4641361 12.50497,14.7200818 12.3063462,14.8710996 C12.1077224,15.0221175 11.8444149,15.0420199 11.6271221,14.9224397 L8.30024457,13.0915213 C8.1122327,12.9882797 7.8877673,12.9882797 7.69975543,13.0915213 L4.37287788,14.9224397 C4.15558509,15.0420199 3.8922776,15.0221175 3.69365379,14.8710996 C3.49502998,14.7200818 3.3955326
```

Comment:
```suggestion
          <td class="table-col-date">
```

If you have to update the flex value here, you have to update it for every row. Probably better to give a class to this that can be updated in one place. The additional advantage is that you can then apply further styles to cells in that column.